include/circt/Dialect/FIRRTL/FIRRTLStructure.td

```diff
@@ -130,6 +130,7 @@ def FModuleOp : FIRRTLModuleLike<"module", [SingleBlock, NoTerminator]> {
   }];
   let arguments =
       (ins ConventionAttr:$convention,
+           DenseBoolArrayAttr:$portDirections,
```

Comment:
It's weird that this _wasn't_ in the argument list before.

Reply:
Nothing was, basically.  Added most of the things last week, but apparently missed this one.